package.json

```diff
@@ -16,8 +16,8 @@
     "build:dev": "webpack --mode development --https false",
     "build-dev": "webpack --mode development --https false && echo . && echo . && echo . && echo Please use 'build:dev' instead of 'build-dev'.",
     "dev-server": "webpack-dev-server --mode development",
-    "lint": "eslint '**/*.{js,jsx}'",
-    "lint:fix": "eslint '**/*.{js,jsx}' --fix",
+    "lint": "eslint **/*.{js,jsx}",
```

Comment:
This is not related to the PR so it is better to make a separate PR for it.

Reply:
@algomaster99 Sure!, I will reopen #47 and make these changes there :)

Reply:
I had made these changes in #51